go-dlv.el

```diff
@@ -131,6 +134,27 @@ and source-file directory for your debugger."
   (setq paragraph-start comint-prompt-regexp)
   (run-hooks 'go-dlv-mode-hook))
 
+;;;###autoload
+(defun dlv-current-func ()
+  "Debug the current program or test stopping at the beginning of the current function."
+  (interactive)
+  (setq test-name nil)
```

Comment:
Enclose the body with `(let ((test-name)) ...`, otherwise this variable leaks into the global namespace.
`current-test-name` might be a better name.